src/app/screens/LNURLWithdraw.tsx

```diff
@@ -61,19 +61,26 @@ function LNURLWithdraw(props: Props) {
 
   function renderAmount() {
     if (minWithdrawable === maxWithdrawable) {
-      return <p>{`${minWithdrawable / 1000} sats`}</p>;
+      return (
+        <ContentMessage
+          heading={`Amount (Satoshi)`}
+          content={`${minWithdrawable / 1000} sats`}
+        />
+      );
     } else {
       return (
-        <div className="mt-1 flex flex-col">
-          <Input
+        <dl className="my-4 p-4 shadow bg-white dark:bg-surface-02dp rounded-lg overflow-hidden">
```

Comment:
`dl` is not valid html with these child-components:
<img width="1163" alt="image" src="https://user-images.githubusercontent.com/1016218/184465443-b82f162b-d02a-46a2-b993-59b8f6e4bccd.png">
(plus some other issues)

Using a `div` here might solve the issue.

Reply:
Changed to div :+1: